timewsync/__init__.py

```diff
@@ -254,8 +271,9 @@ def _generate_key(data_dir: str) -> None:
     Args:
         data_dir: The user's timewsync data dir.
     """
-    log = logging.getLogger(__name__)
+    log = logging.getLogger("timewsync")
```

Comment:
What's in `__name__` by default? It looks like the docs recommend initializing the logger with `__name__`:

https://docs.python.org/3/library/logging.html#logger-objects

Reply:
The main idea behind the hardcoded string was to provide a logger object throughout the entire application.

I just tested it a bit:
- in `__init__.py`, `__name__` returns `timewsync`
- in `io_handler.py`, `__name__` returns `timewsync.io_handler`

Since the logger `timewsync.io_handler` is a child of our main logger, I don't see any problem in setting it back to `__name__` as you somewhat suggested.